aws_lambda/s3_create_datasource_in_db.py

```diff
@@ -37,11 +38,23 @@ def lambda_handler(event, context):
             if key.endswith('/'):  # if key is folder
                 continue
 
+            # determine the mime type for the file
+            mimetypes.init()
+            # adding two mime types for excel spreadsheet
+            mimetypes.add_type("application/vnd.ms-excel", ".xls", strict=True)
+            mimetypes.add_type("application/vnd.openxmlformats-officedocument.spreadsheetml.sheet", ".xlsx", strict=True)
+            # extract mime type from file name
+            mime_type = mimetypes.guess_type(uri)[0]
```

Comment:
It's better to place these lines outside of the function. So they wouldn't be called each lambda invocation.